.github/workflows/compute.yml

```diff
@@ -18,12 +18,12 @@ on:
 
 jobs:
   compute:
-    name: "plugin name"
+    name: "Telegram-Bot"
     runs-on: ubuntu-latest
     permissions: write-all
     env:
-      SUPABASE_URL: ${{ secrets.SUPABASE_URL }}
-      SUPABASE_KEY: ${{ secrets.SUPABASE_KEY }}
+      TELEGRAM_BOT_ENV: ${{ secrets.TELEGRAM_BOT_ENV }}
+      REPOSITORY: ${{ secrets.REPOSITORY }}
```

Comment:
What is this? You need to include prefixes for context, like telegram

Reply:
All env vars are handled via `yarn setup-env` but it's this repo' `full_name`

Reply:
Why do you need the repo name? Can't this be dynamically read?

Reply:
I don't think so because Context will always have the payload repo. We need it for `repo_dispatch` and we pass in `targetBranch` via the config as opposed to env vars, making the repo immutable but the branch configurable.